zrml/swaps/src/benchmarks.rs

```diff
@@ -163,12 +172,72 @@ benchmarks! {
                 status: MarketStatus::Active,
             }
         )?;
-        let _ = T::MarketCommons::insert_market_pool(0u32.saturated_into(), 0u128);
-        let _ = bench_create_pool::<T>(caller, Some(T::MaxAssets::get().into()), None, ScoringRule::CPMM, false);
-        let _ = Pallet::<T>::mutate_pool(0, |pool| {
-            pool.pool_status = PoolStatus::Closed; Ok(())
+        let market_id = T::MarketCommons::latest_market_id()?;
```

Comment:
The `market_id` is returned by `push_market` above:
```rust
    /// Pushes a new market into the storage, returning its related auto-incremented ID.
    fn push_market(
        market: Market<Self::AccountId, Self::BlockNumber, Self::Moment>,
    ) -> Result<Self::MarketId, DispatchError>;
```

We should use the market id retrieved by that instead of doing an additional storage read. With the current default benchmark configuration each benchmark is executed up to 10,000 times, so a storage operation can become quite heavy in total.

Reply:
Mitigated in 643812487ea202067721351d6381c38c699dd910